installer/x86_64/install.sh

```diff
@@ -486,29 +514,42 @@ fi
 # Add a menu entry for the DEMO OS
 # Note: assume that apparmor is supported in the kernel
 demo_grub_entry="$demo_volume_revision_label"
+if [ "$install_env" = "sonic" ]; then
+    old_sonic_menuentry=$(cat /host/grub/grub.cfg | sed "/$running_sonic_revision/,/}/!d")
+    demo_dev=$(echo $old_sonic_menuentry | sed -e "s/.*root\=\(.*\)rw.*/\1/")
+    onie_menuentry=$(cat /host/grub/grub.cfg | sed "/menuentry ONIE/,/}/!d")
+fi
+
 cat <<EOF >> $grub_cfg
 menuentry '$demo_grub_entry' {
         search --no-floppy --label --set=root $demo_volume_label
-        echo    'Loading $demo_volume_revision_label $demo_type kernel ...'
+        echo    'Loading $demo_volume_label $demo_type kernel ...'
```

Comment:
why not keep the demo_volume_revision_label here? We need to know which version we are loading.